data/json/vehicle_placement.json

```diff
@@ -0,0 +1,107 @@
+[
+    {
+        "type" : "vehicle_placement",
+        "id" : "road_straight_wrecks",
+        "locations" : [
+            { "x": [0,19], "y": [0,19], "facing" : [90,270] }
```

Comment:
Can't really put it with the mapgen json, because it is for mapgens that don't have any json ;)